src/main/java/com/example/mybatis0603/NameMapper.java

```diff
@@ -11,6 +11,5 @@ public interface NameMapper {
     @Select("SELECT * FROM anime")
     List<Name> findAll();
     @Select("SELECT * FROM anime WHERE id = #{id}")

```

Comment:
メソッド名ごとに1行あけるのがベターです。
```suggestion
    @Select("SELECT * FROM anime")
    List<Name> findAll();

    @Select("SELECT * FROM anime WHERE id = #{id}")

```